ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2

```diff
@@ -356,10 +345,10 @@ host_monitoring_cron:
   job: ops-runner -f -s 300 -n ozc.send.synthetic.heartbeat.full /usr/bin/ops-metric-client --send-heartbeat --synthetic
 
 - name: Do a quick heartbeat for synthetic host
-  minute: "*"
+  minute: "*/2"
```

Comment:
This should be in sync with the normal heartbeat, which is set to every 3 minutes.

Sorry, I probably screwed up and said 2 here.